lib/formatters/grep.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview grep-style formatter.
+ * @author Nicholas C. Zakas
```

Comment:
Is it necessary? I am already telling you my real name in the CLA so you've got the copyright covered.